.agent/plans/social-community-features.md

```
@@ -0,0 +1,523 @@
+│   │   │   └── trophy-showcase.tsx
+│   │   └── hooks/
+│   │       ├── use-leaderboard.ts
+│   │       ├── use-user-score.ts
+│   │       └── use-user-trophies.ts
+│   └── ratings/components/
+│       └── rating-like-button.tsx
+```
+
+**Updated Files**:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)